### PR TITLE
Added ResourceClassNameId attribute for the ADTs

### PR DESCRIPTION
**Pull request by jsgomezp**

### CODE REVIEW

File: db-setup-core/src/main/java/com/ableneo/liferay/portal/setup/core/SetupArticles.java [hunk 1]
```diff
@@ -4,8 +4,7 @@
  * #%L
  * Liferay Portal DB Setup core
  * %%
- * Original work Copyright (C) 2016 - 2018 mimacom ag
- * Modified work Copyright (C) 2018 - 2020 ableneo, s. r. o.
+ * Copyright (C) 2016 - 2019 ableneo s. r. o.
```
Comment (ktor) [inline db-setup-core/src/main/java/com/ableneo/liferay/portal/setup/core/SetupArticles.java]: Hi jsgomezp, thank you for the pull request, please leave the copyright intact.

Comment (jsgomezp) [reply db-setup-core/src/main/java/com/ableneo/liferay/portal/setup/core/SetupArticles.java]: Reverted those lines. Didn't notice they changed.

**Comment by ktor** [comment]
Hi @jsgomezp , build's still failing.

**Comment by ktor** [comment]
Hi @jsgomezp build was failing because sonar configuration is broken for pull requests. It needs a fix.

**Pull request merged**